pkg/operation/botanist/component/resourcemanager/resource_manager.go

```diff
@@ -754,6 +758,11 @@ func (r *resourceManager) computeCommand() []string {
 	if r.values.TargetDiffersFromSourceCluster {
 		cmd = append(cmd, "--target-kubeconfig="+gutil.PathGenericKubeconfig)
 	}
+	if r.values.SchedulingProfile != nil && *r.values.SchedulingProfile != gardencorev1beta1.SchedulingProfileBalanced {
+		cmd = append(cmd, "--pod-scheduler-name-webhook-enabled=true")
+		cmd = append(cmd, fmt.Sprintf("--pod-scheduler-name-webhook-scheduler=%s", string(*r.values.SchedulingProfile)+"-scheduler"))
```

Comment:
Here you could now use the constant that you introduced based on earlier suggestion.